core/appmodule/module.go

```diff
@@ -15,16 +14,22 @@ import (
 // by other modules (usually via depinject) as app modules.
 type AppModule = appmodule.AppModule
 
-// HasMigrations is the extension interface that modules should implement to register migrations.
-type HasMigrations interface {
-	AppModule
+// HasPreBlocker is the extension interface that modules should implement to run
+// custom logic before BeginBlock.
+// It can modify consensus parameters in storage and signal the caller through the return value.
```

Comment:
`// It can modify consensus parameters in storage and signal the caller through the return value.`

This is not like this anymore, should be removed right?